source/features/comments-time-machine-links.tsx

```diff
@@ -71,6 +71,11 @@ async function showTimemachineBar(): Promise<void | false> {
 			return false;
 		}
 
+		const lastCommitDate = await elementReady('.repository-content .Box.Box--condensed relative-time');
```

Comment:
Not compatible with the repository refresh. You can probably **add** `[itemprop="dateModified"] relative-time`. Add a JS comment exactly like in the other "Repository refresh" commits.

Reply:
Hope you like the way I did the comment. It's a bit different here since it's just another selector.